packages/ui/src/components/TextArea.tsx

```diff
@@ -29,8 +29,8 @@ export function TextArea<T extends FieldValues>({
       placeholder={placeholder}
       {...register(name)}
       className={clsx(
-        "form-input block w-full text-sm shadow-sm rounded-md border-gray-300 focus:ring-indigo-500 active:ring-indigo-500 focus:border-indigo-500 active:border-indigo-500",
-        disabled && "text-gray-400"
+        "form-input block w-full text-sm shadow-sm rounded-md border-slate-300 focus:ring-indigo-500 active:ring-indigo-500 focus:border-indigo-500 active:border-indigo-500 placeholder:text-slate-300",
```

Comment:
these changes seem safe, though a bit unrelated from the key pr

Reply:
Yeah, the default placeholder color was too dark and it was confusing.